testcases/realtime/run.sh

```diff
@@ -203,13 +203,17 @@ do
 	case "$option" in
 
 	t )
+		LOOP=1
 		if [ $ISLOOP -eq 1 ]; then
-			LOOP=1
 			tests[$index]=$LOOP
 			: $(( index += 1 ))
 		fi
 
 		tests[$index]="$OPTARG"
+		if [ $index -eq 0 ]; then
```

Comment:
I'll have to look at the call order/how it feeds it into find_test, but this seems a bit obfuscated at first glance (in part because the variable names aren't verbose).